apps/backend/src/services/accounts.js

```diff
@@ -176,137 +172,141 @@ async function hydrateDemoData({ userId }) {
         ],
       },
     },
-    false,
-    {
+  },
+  {
+    title: 'Demoquestion FT',
+    type: 'FREE',
+    content: 'Describe a main principle of a social market economy.',
+    options: false,
+  },
+  {
+    title: 'Demoquestion NR',
+    type: 'FREE_RANGE',
+    content: 'Estimate the length of the **longest** river in the world (answer in kilometres).',
+    options: {
       FREE_RANGE: {
-        randomized: false,
         restrictions: { min: 0, max: 8000 },
-        choices: [],
       },
     },
-  ]
-  const demoQuestions = []
+  },
+]
+
+function prepareDemoSessionData({ id, name, user, blockInstances, blockStatus = 'PLANNED', ...rest }) {
+  return new SessionModel({
+    _id: id,
+    settings: {
+      isParticipantAuthenticationEnabled: false,
+      isConfusionBarometerActive: true,
```

Comment:
Changed the default here so the confusion barometer is active immediately on the started demo session